setup_guide.md

```diff
@@ -0,0 +1,91 @@
+# How to install and run IMPRESS / EXACT / IMPISH flight code
+
+This guide will show you how to setup the flight code on a fresh pi.
+
+## Table of Contents
+1. [Installation](#installation)
+2. [Usage](#usage)
+3. [PPS Setup](#pps-setup)
+4. [Data Analysis](#data-Analysis)
+
+
+## Installation
+
+First install necessary libraries:
+```
+sudo apt install libusb-1.0-0-dev gpiod libgpiod-dev libboost-all-dev libgtest-dev libsystemd-dev
+```
+
+Install git and python/pip:
+```
+sudo apt install git
+sudo apt install python3
+```
+
+Clone the flight code repository:
+```
+git clone https://github.com/umn-impish/umn-detector-code.git
+```
+
+Cd into the flight code directory and compile the code:
+```
+cd ~/umn-detector-code/flight-controller
+mkdir build && cd build
+cmake ..
+make -j4
+```
+
+You will also have to move the usb.rules file into /etc/udev/rules.d/. This will give all usb devices read write access to the pi.
+```
+sudo mv ~/umn-detector-code/flight-controller/install/usb.rules /etc/udev/rules.d/
```

Comment:
i think we should change this so that it only adds the Amptek and Bridgeport devices to the usb.rules... otherwise if people are testing on their local machine, they may have some kind of weird security vulnerability introduced

Reply:
also remove the `~`

Reply:
the ID's required for permissions might be unique for each board, but I'll look into it

Reply:
you should just need the vendor ID and product ID. they are in the USB code portions or you can get them from lsusb